.travis.yml

```diff
@@ -7,8 +7,8 @@ node_js:
 cache: yarn
 
 script:
-  - yarn run lint
-  - yarn run test:coverage --runInBand
+  - yarn lint
+  - yarn test:coverage --runInBand
```

Comment:
@dcyriller why is this changing now? This should be in a separate PR if we want to change it.